edgeless_api/src/node_registration.rs

```diff
@@ -23,7 +23,7 @@ pub struct NodeCapabilities {
 }
 
 impl NodeCapabilities {
-    pub fn default() -> Self {
```

Comment:
https://doc.rust-lang.org/std/default/trait.Default.html
Note that there is a default trait that is commonly used.
That would not provide the required semantics here.

Reply:
Thanks, I did not know that (before running clippy). I changed the name, instead of implementing the trait, precisely because the semantics is different.